src/analyses/constant_propagator.cpp

```diff
@@ -74,6 +74,7 @@ void constant_propagator_domaint::transform(
   output(std::cout, ai, ns);
 #endif
 
+  #if 0
```

Comment:
Is this code going to come back again? If so add a comment saying how and when, otherwise remove it totally and let git keep the history for future readers (multiple occurrences).